ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -142,6 +142,12 @@ def argmax(self, axis=None, keepdim=False, dtype=None, name=None):
             ivy.argmax(self._ivy_array, axis=axis, keepdims=keepdim, dtype=dtype)
         )
 
+    @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+    def lerp(self, y, weight, name=None):
+        return paddle_frontend.Tensor(
+            ivy.lerp(self._ivy_array, _to_ivy_array(y), weight)
+        )

```

Comment:
There is already a `lerp` function in the Paddle frontend. You should use that here like so:
```python
return paddle_frontend.lerp(self, y, weight)
```